README.md

```diff
@@ -53,10 +53,10 @@ which is handy when working on a particular exercise:
     ./sbt
 
     # Run the tests from a package containing the name "Id"
-    > testOnly "*Id*"
+    > testOnly *Id*
```

Comment:
For some reason testOnly wasn't working when these things were quoted.

Reply:
Yeah not sure why I put the quotes in 👍